project/src/index.tsx

```diff
@@ -5,7 +5,7 @@ import App from './components/app/app';
 import {Provider} from 'react-redux';
 import {store} from './store';
 import {checkAuthAction, fetchOffersAction} from './store/api-actions';
-import ErrorMessage from './components/error-message/error-message';
+import {ToastContainer} from 'react-toastify';
```

Comment:
```suggestion
import {ToastContainer} from 'react-toastify';

import 'react-toastify/dist/ReactToastify.css';
```
Нужно для него ещё стили импортировать.